db/migrate/20230727072004_create_keywords.rb

```diff
@@ -0,0 +1,26 @@
+class CreateKeywords < ActiveRecord::Migration[7.0]
+  def change
+    create_enum  :keyword_status, %w[processing succeeded failed]
+
+    create_table :keywords, id: :serial do |t|
+      t.string :content, limit: 255, null: false
+      t.enum :status, enum_type: 'keyword_status', default: 'processing', null: false
```

Comment:
```suggestion
      t.enum :status, enum_type: 'keyword_status', default: 'pending', null: false
```

Reply:
Oops, git actions gone rogue here. Fixed in 91c28f7a80275a762abfc0bb70c22db03b15c423